src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search__title--big">
+      <div
+        class="search__link--big"
+        data-qa="big"
+      >
+        <input
+          class="search__big"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </div>
+    </div>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search__title--small">
+      <div
+        class="search__link--small"
+        data-qa="small"
```

Comment:
Similar to the previous comment, the `div` element here is also missing a closing tag. Make sure to close all `div` elements to avoid structural issues.